hawkbit-rest/hawkbit-ddi-api/src/main/java/org/eclipse/hawkbit/ddi/rest/api/DdiRootControllerRestApi.java

```diff
@@ -755,7 +755,7 @@ ResponseEntity<Void> postConfirmationActionFeedback(@Valid final DdiConfirmation
         be automatically confirmed, as long as auto-confirmation is active.
         """)
     @ApiResponses(value = {
-        @ApiResponse(responseCode = "200", description = "Successfully retrieved"),
+        @ApiResponse(responseCode = "201", description = "Successfully created"),
```

Comment:
If you check the implementation here activateAutoConfirmation returns 200 : 

return new ResponseEntity<>(HttpStatus.OK);